build-test.cmd

```diff
@@ -22,15 +26,18 @@ set __BuildOS=Windows_NT
 :: is already configured to use that toolset. Otherwise, we will fallback to using the VS2015
 :: toolset if it is installed. Finally, we will fail the script if no supported VS instance
 :: can be found.
-if defined VisualStudioVersion goto :Run
+if defined VisualStudioVersion ( 
+    if not defined __VSVersion echo %__MsgPrefix%Detected Visual Studio %VisualStudioVersion% developer command ^prompt environment
```

Comment:
Why the `^`  in front of prompt?

Reply:
prompt is detected as keyword - ^ put in front escapes it and is not visible when printed out